src/app/projects/project-detail/documents/documents-tab-content.component.spec.ts

```diff
@@ -37,10 +37,28 @@ describe('DocumentsTabContentComponent', () => {
     component = fixture.componentInstance;
     component.collections = new CollectionsArray();
   });
-
-  it('should be created', () => {
-    spyOn(component, 'parseData').and.stub;
-    fixture.detectChanges();
-    expect(component).toBeTruthy();
+  describe('ngOnInit()', () => {
+    beforeEach(() => {
+      component.ngOnInit();
+    });
+    it('should return data for route.data', () => {
+      expect(ActivatedRouteStub.data).toBeTruthy;
+    });
+    it('should return project data', () => {
+      expect(component.project).toBeTruthy;
+    });
+  });
+  describe('parseData(data)', () => {
+    beforeEach(() => {
+      const data = { project: new Project() };
+      data.project.collections = new CollectionsList();
+      component.parseData(data);
+    });
+    it('should return data this.project', () => {
```

Comment:
nit: 'should return data for this.project'

Reply:
done